.github/workflows/continuous-integration-cypress.yml

```diff
@@ -1,15 +1,19 @@
 name: Cypress tests
 
 on:
-  push:
+  workflow_run:
+    workflows: ["Deploy to environment (CIP)"]
     branches: [ main ]
+    types:
+      - completed
```

Comment:
@jordan-beard this is a (relatively) new feature of Github actions but you can now specify a workflow dependancy so that this workflow only runs if the specified one completes.

Reply:
@jordan-beard currently, that workflow allows you to deploy to any env (Dev,Test,Prod) by invoking it manually but we can split off deployment to Development so its fully automated (i.e. doesn't require a manual invoke) if you prefer?

Reply:
Yes please.
The way it works at the moment -
- When we merge to main a deployment to dev is triggered automatically which generates us a dev tag.
- When we've tested it on dev and want to push to test we manually trigger deploy to test (from dev tag).
- When we've tested it on test and want to push to prod we manually trigger deploy to prod (from test tag).

If its ok I'd like to stick as close to this deployment workflow as possible for the new Azure deployment.